docker-compose.yml

```diff
@@ -15,10 +15,9 @@ services:
     expose:
       - 9000
     volumes:
-      - .:/var/www/html
-      - composer:/var/www/html/vendor
-      - runtime:/var/www/html/storage/runtime
-      - logs:/var/www/html/storage/logs
+      - .:/var/www/html:cached
+      - ./vendor:/var/www/html/vendor:delegated
+      - ./storage/logs:/var/www/html/storage/logs:delegated
```

Comment:
Came across this in our #dev slack channel a couple of days ago. 

Apparently `delegated` is no more on newer Docker for Mac? 

https://github.com/docker/for-mac/issues/5402